From 6fff15f12cd2cc613456f30e40d0b3684138a609 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Wed, 1 Apr 2026 01:30:37 +0100
Subject: [PATCH] update unneeded-requirements.patch

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index b22c74c9..939c9304 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -28,7 +28,6 @@ dependencies = [
 	"corner",
 	"gwpy >=3.0.11",
 	"h5py",
-	"lalsuite >7.11",
 	"ligo-gracedb",
 	"matplotlib >= 3.10.0",
 	"numpy >=1.15.4,<2.4.0", # 2.4.0 incompatible with astropy
-- 
2.50.1 (Apple Git-155)

