From 3a56c4892e23d6411cd3f850640cbaeff88b2755 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Mon, 24 Apr 2023 09:32:11 +0100
Subject: [PATCH] patch pyproject.toml for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index dc3332b5..e35d88f2 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,7 +30,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=2.1.2",
 	"h5py",
-	"lalsuite >=7.0.0",
 	"ligo-gracedb",
 	"matplotlib",
 	"numpy >=1.15.4",
-- 
2.37.0 (Apple Git-136)

