From b74d4ed90075afbc9cab03588966acc735333078 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Tue, 20 Feb 2024 12:20:32 +0000
Subject: [PATCH] remove lalsuite for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 4b0ac5dd..b49aa901 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=2.1.2",
 	"h5py",
-	"lalsuite >7.11",
 	"ligo-gracedb",
 	"matplotlib<3.8.0",
 	"numpy >=1.15.4",
-- 
2.37.0 (Apple Git-136)

