From 2d609d8ab824de07700c07d50806a25c6ba6abd7 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Mon, 23 Oct 2023 18:04:17 +0100
Subject: [PATCH] remove lalsuite for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 7808b328..31b8f010 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=2.1.2",
 	"h5py",
-	"lalsuite >=7.0.0",
 	"ligo-gracedb",
 	"matplotlib<3.8.0",
 	"numpy >=1.15.4",
-- 
2.37.0 (Apple Git-136)

