From 7b26257a7472acda12908fd703d664c511cab64c Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Fri, 19 Jan 2024 10:31:21 +0000
Subject: [PATCH] remove lalsuite for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 7808b328..31b8f010 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=2.1.2",
 	"h5py",
-	"lalsuite >=7.0.0",
 	"ligo-gracedb",
 	"matplotlib<3.8.0",
 	"numpy >=1.15.4",
-- 
2.37.0 (Apple Git-136)

