From 989e1fe114495517fbdb4f2619293f6429dff357 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Mon, 1 Jul 2024 23:47:38 +0100
Subject: [PATCH] remove lalsuite for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 9c790bed..a6282c09 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=2.1.2",
 	"h5py",
-	"lalsuite >7.11",
 	"ligo-gracedb",
 	"matplotlib<3.8.0",
 	"numpy >=1.15.4",
-- 
2.37.0 (Apple Git-136)

