From 23b88b7656ef33238e782c7c8fd2eeb3612eeb77 Mon Sep 17 00:00:00 2001
From: Charlie Hoy <charlie.hoy@port.ac.uk>
Date: Thu, 3 Jul 2025 08:49:00 +0100
Subject: [PATCH] remove lalsuite for conda

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0b388ce6..f6e6898c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,6 @@ dependencies = [
 	"deepdish",
 	"gwpy >=3.0.11",
 	"h5py",
-	"lalsuite >7.11",
         "python-ligo-lw <2.0.0",
 	"ligo-gracedb",
 	"matplotlib >= 3.6.0,<3.10.0",
-- 
2.39.3 (Apple Git-146)

